modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/upgrade/v2_13_1/CTConflictConfigurationUpgradeProcess.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.change.tracking.internal.upgrade.v2_13_1;
+
+import com.liferay.change.tracking.configuration.CTSettingsConfiguration;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.configuration.module.configuration.ConfigurationProvider;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.util.Dictionary;
+
+import org.osgi.service.cm.Configuration;
+import org.osgi.service.cm.ConfigurationAdmin;
+
+/**
+ * @author David Truong
+ */
+public class CTConflictConfigurationUpgradeProcess extends UpgradeProcess {
+
+	public CTConflictConfigurationUpgradeProcess(
+		ConfigurationAdmin configurationAdmin,
+		ConfigurationProvider configurationProvider) {
+
+		_configurationAdmin = configurationAdmin;
+		_configurationProvider = configurationProvider;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		Configuration oldConfiguration = _configurationAdmin.getConfiguration(
+			_CLASS_NAME_CTCONFLICT_CONFIGURATION, StringPool.QUESTION);
```

Comment:
CTConflictConfiguration is a company scoped configuration so there can be multiple instances of the configuration. We should get all the configurations
```
Configuration[] configurations = _configurationAdmin.listConfigurations(
	String.format(
		"(%s=%s.scoped)", ConfigurationAdmin.SERVICE_FACTORYPID,
		_CLASS_NAME_CTCONFLICT_CONFIGURATION));
``` 

After that, iterate through the list to update each configuration. Note that `listConfigurations` can return null, so we might need to do a null check to avoid NPE.